src/system.ts

```diff
@@ -472,15 +472,15 @@ metrics.registerQueueLengthGauge('ans104Unbundler', {
   length: () => ans104Unbundler.queueDepth(),
 });
 
-export const bundleDataImporter = new BundleDataImporter({
+export const dataImporter = new DataImporter({
```

Comment:
The `bundelDataImporter` name should stay the same. Instead we should be instantiating a separate `DataImporter` that uses a chunk based contiguous data source and pass that to the verification worker.

Reply:
I see, that's a good idea, I will adjust. If having two sepearate classes would make sense too if the logic stats to diverge greatly enough. But I keep it 1 and same class as is called for, for now.

Reply:
done

Reply:
> If having two sepearate classes would make sense too if the logic stats to diverge greatly enough. But I keep it 1 and same class as is called for, for now.

Agree. I think we're good on that front for now, but if we get more conditionals in there we'll want to think about it more.